routes/user.js

```diff
@@ -213,9 +210,9 @@ router.route('/forgot')
                     text: 'Someone has requested to reset the password for your account at glorify.cc\n\n' +
                         'If you did not request this, you can ignore this message.\n\n' +
                         'Otherwise, please follow this link to reset your password:\n' +
-                        'http://' + req.headers.host + '/user/reset/' + token
+                        (process.env.NODE_ENV ? 'https' : 'http') + '://' + req.headers.host + '/user/reset/' + token
```

Comment:
you need to compare it with 'production' because if it is 'development' you still want to use http probably.